wd.sh

```diff
@@ -472,6 +497,10 @@ else
                 wd_browse
                 break
                 ;;
+            "-c"|"--addcd"|"addcd")
+                wd_addcd "$2" "$3" "$wd_force_mode"
```

Comment:
I forget how Bash works - will this error if `$3` is unset or does it just become empty string?

Reply:
Empty string